src/landing-page/components/LandingPage/index.tsx

```diff
@@ -0,0 +1,301 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Button, Tooltip } from "antd";
+import React, { ReactElement } from "react";
+
+import { landingPageContent } from "./content";
+import { DatasetEntry, ProjectEntry, ViewerArgs } from "../../types";
+import styled from "styled-components";
+import { FlexColumnAlignCenter, FlexColumn, FlexRowAlignCenter, VisuallyHidden, FlexRow } from "./utils";
+
+const Banner = styled(FlexColumnAlignCenter)`
+  position: relative;
+  --container-padding-x: 20px;
+  padding: 30px var(--container-padding-x);
+  overflow: hidden;
+  margin: 0;
+`;
+
+const BannerTextContainer = styled(FlexColumn)`
+  --padding-x: 30px;
+  padding: var(--padding-x);
+  max-width: calc(1060px - 2 * var(--padding-x));
+
+  --total-padding-x: calc(2 * var(--padding-x) + 2 * var(--container-padding-x));
+  width: calc(90vw - var(--total-padding-x));
+  border-radius: 5px;
+  // Fallback in case color-mix is unsupported.
+  background-color: var(--color-background);
+  // Make the background slightly transparent. Note that this may fail on internet explorer.
+  background-color: color-mix(in srgb, var(--color-background) 80%, transparent);
+  gap: 10px;
+
+  & > h1 {
+    margin-top: 0;
+  }
+
+  & > p {
+    font-size: var(--font-size-label);
+  }
+`;
+
+const BannerVideoContainer = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #ded9ef;
+  z-index: -1;
+
+  & > video {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    // Fixes a bug where a single pixel black outline would appear around the video.
+    clip-path: inset(1px 1px);
+  }
+`;
+
+const ContentContainer = styled(FlexColumn)`
+  max-width: 1060px;
+  width: calc(90vw - 40px);
+  margin: auto;
+  padding: 0 20px;
+`;
+
+const FeatureHighlightsContainer = styled.li`
+  display: grid;
+  width: 100%;
+  grid-template-rows: repeat(2, auto);
+  grid-template-columns: repeat(auto-fit, minmax(230px, 1fr));
+  padding: 0;
+  justify-content: space-evenly;
+  gap: 10px;
+  margin: 20px 0;
+`;
+
+const FeatureHighlightsItem = styled(FlexColumn)`
+  display: grid;
+  grid-template-rows: subgrid;
+  grid-row: span 2;
+
+  & > h3 {
+    font-weight: 600;
+  }
+`;
+
+const Divider = styled.hr`
+  display: block;
+  width: 100%;
+  height: 1px;
+  background-color: var(--color-borders);
+  border-style: none;
+`;
+
+const ProjectList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 0;
+  margin-top: 0;
+
+  // Add a pseudo-element line between cards
+  & > li:not(:first-child)::before {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 1px;
+    background-color: var(--color-borders);
+    margin-bottom: 10px;
+  }
+`;
+
+const ProjectCard = styled.li`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  gap: 12px;
+
+  & h3 {
+    font-weight: 600;
+  }
+`;
+
+const DatasetList = styled.ol`
```

Comment:
why are you using an ordered list here?

Reply:
Looking into this, I think an unordered list makes more sense, yeah. 

This is what the lists of datasets look like in the colorizer tool: 
![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/8bf32bea-038e-4409-9695-86fa60b613fc)